### PR TITLE
People 164 order skills by name

### PR DESCRIPTION
**Pull request by DariuszMusielak**
# Ticket
https://netguru.atlassian.net/browse/PEOPLE-164

# Description
- sort skills by name and by skill category name 

### Before merge checklist
- [ ] CircleCI is passing
- [ ] Proper labels are set

### CODE REVIEW

File: app/queries/user_skill_rates_query.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@ def initialize(user)
   end
 
   def results_by_categories
-    user_skill_rates.group_by { | skill | skill.category }
+    user_skill_rates.group_by do |skill|
+      skill.category
+    end.sort_by{ |key, _| key }.to_h
```
Comment (netguru-hound) [inline app/queries/user_skill_rates_query.rb]: Avoid multi-line chains of blocks.<br>Space missing to the left of {.

File: app/queries/user_skill_rates_query.rb [hunk 2]
```diff
@@ -6,7 +6,9 @@ def initialize(user)
   end
 
   def results_by_categories
-    user_skill_rates.group_by { | skill | skill.category }
+    user_skill_rates.group_by do |skill|
```
Comment (netguru-hound) [inline app/queries/user_skill_rates_query.rb]: Pass &:category as an argument to group_by instead of a block.

Comment (elharion) [inline app/queries/user_skill_rates_query.rb]: this :point_up:

Comment (elharion) [inline app/queries/user_skill_rates_query.rb]:
```ruby
.group_by(&:category).sorty_by{ |key, _| key }.to_h
```

File: app/queries/user_skill_rates_query.rb [hunk 3]
```diff
@@ -6,7 +6,7 @@ def initialize(user)
   end
 
   def results_by_categories
-    user_skill_rates.group_by { | skill | skill.category }
+    user_skill_rates.group_by(&:category).sort_by{ |key, _| key }.to_h
```
Comment (netguru-hound) [inline app/queries/user_skill_rates_query.rb]: Space missing to the left of {.

**Pull request merged**